bin/rad-diff

```diff
@@ -207,31 +199,22 @@
     (def mk-key
       (fn [v]
         (match v
-          (/as 'a {:activity :new-diff
-                   :diff     'diff})
-            [(string-append "[new/changed] " (pretty-headline diff)) a]
-          (/as 'a {:activity :new-comments
-                   :diff     'diff})
-            [(string-append "[new comments] " (pretty-headline diff)) a]
           (/as 'i {:diff 'diff})
             [(pretty-headline diff) i])))
 
     (def filt
-      (match (filter (fn [opt] (not (elem? opt [:new :fancy]))) options)
+      (match (filter (fn [opt] (not (eq? opt :fancy))) options)
         []      (fn [x] #t)
         _       (fn [x] (elem? (lookup :state (lookup :diff x)) options))))
 
   (def items
     (map mk-key
       (filter filt
-        (match [(elem? :new options) (read-file-key! ".radicle-diffs" :last-mark-done)]
-           [#t [:just 't]] (recent-activity chain-ref t)
-           _               (map (fn [i] {:diff i})
-                                (reverse (values (list-diffs chain-ref))))))))
+        (map (fn [i] {:diff i}) (reverse (values (list-diffs chain-ref)))))))
```

Comment:
I think we can remove the wrapping of the diff in `{:diff i}`. This would eliminate all the `:diff` lookups and matches against `diff`.